src/lib/components/PromptWithSlider.svelte

```diff
@@ -0,0 +1,145 @@
+<script lang="ts">
+	import type { Section } from '$lib/types';
+
+	export let value: number;
+	export let section: Section;
+	export let index: number;
+	export let scrollToSection: (evt: Event, idx: number) => void;
```

Comment:
nit: since this component is agnostic of what the action should do, maybe we update the name of this function to `handleSectionChange` or the like?

Reply:
sounds great, just updated!